src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -15,9 +15,14 @@ export type FileUploadTemplateProps = {
   /**
    * Whether the file input accepts multiple files.
    * Passed to the `multiple` attribute of native file input.
-   * @deprecated Use `inputProps` instead
+   * @default true
```

Comment:
maybe leave it as false..

Reply:
oh wait, it was true by default.. I just checked the last commit, so thought its added as new one...

Reply:
"leave"? It was `true` before this PR.

I had removed the default value because of the deprecation so I'm just reverting that